src/mame/palm/palm.cpp

```diff
@@ -27,8 +28,6 @@
 #define VERBOSE (0)
 #include "logmacro.h"
 
-namespace {
-
```

Comment:
I’m assuming the reason this was taken out was because there was a local device here, and device type definitions need to be in the global namespace.  However, that doesn’t seem to be the case now – can it go back?

Reply:
Re-added, thanks for the reminder.